### PR TITLE
Some couchexport cleanup

### PR DESCRIPTION
**Pull request by dannyroberts**
easiest to review commit by commit

### CODE REVIEW

File: corehq/apps/reports/views.py [hunk 1]
```diff
@@ -427,7 +453,17 @@ def hq_download_saved_export(req, domain, export_id):
 
     export.last_accessed = datetime.utcnow()
     export.save()
-    return couchexport_views.download_saved_export(req, export_id)
+    export = SavedBasicExport.get(export_id)
+    content_type = Format.from_format(export.configuration.format).mimetype
+    payload = export.get_payload(stream=True)
+    response = StreamingHttpResponse(FileWrapper(payload), content_type=content_type)
+    if export.configuration.format != 'html':
+        # ht: http://stackoverflow.com/questions/1207457/convert-unicode-to-string-in-python-containing-extra-symbols
```
Comment (dimagimon) [inline corehq/apps/reports/views.py]: E501 line too long (117 > 115 characters)

File: corehq/apps/reports/views.py [hunk 2]
```diff
@@ -427,7 +453,17 @@ def hq_download_saved_export(req, domain, export_id):
...
+        normalized_filename = unicodedata.normalize(
```
Comment (dimagimon) [inline corehq/apps/reports/views.py]: F821 undefined name 'unicodedata'

**Comment by dannyroberts** [comment]
@czue cc: @nickpell @biyeun

**Comment by dannyroberts** [comment]
I just tested this locally by making sure a default form export and custom form export had the same behavior as on master. (Really I'd expect any error to be a 500/exception, because none of the logic has changed.)

File: corehq/apps/reports/views.py [hunk 3]
```diff
@@ -276,10 +278,33 @@ def export_data_async(request, domain):
     except ValueError:
         return HttpResponseBadRequest()
     assert(export_tag[0] == domain)
+    format = request.GET.get("format", Format.XLS_2007)
+    filename = request.GET.get("filename", None)
+    previous_export_id = request.GET.get("previous_export", None)
 
     filter = util.create_export_filter(request, domain, export_type=export_type)
```
Comment (nickpell) [inline corehq/apps/reports/views.py]: (nonblocking) overrides python function ```filter```

Comment (dannyroberts) [inline corehq/apps/reports/views.py]: Yeah, `format` and `filter` shadow the built-ins—unfortunately this is the pattern all over couchexport so just sticking with that convention for now

**Comment by biyeun** [comment]
:+1: will let @czue have a look

**Comment by czue** [comment]
looks good!

**Pull request merged**